spec/rails_helper.rb

```diff
@@ -1,4 +1,5 @@
 ENV["RAILS_ENV"] ||= "test"
+ENV["ORGANISATION_FILE_PATH"] = "spec/support/organisation_types.yml"
```

Comment:
Yup that would be a better home for it.
In development you would want to use the actual list of organisation types